### ISSUE TITLE
Not an issue: Will there be newer versions with the new features of domino-db 1.0.3+?

### ISSUE THREAD

**Issue by ThomasBahn** [stefanopog/node-red-contrib-dominodb#6]
With 1.0.3 out two months ago the question arises: Will there be versions with the newer features of domino-db, like attachments?

What are the future plans for this project?

**Comment by ba47-development** [comment]
Yes, agree, attachments or rich text would be a great improvement!
